src/Symfony/Component/DependencyInjection/Definition.php

```diff
@@ -830,6 +832,67 @@ public function isAbstract()
     }
 
     /**
+     * Whether this definition is deprecated, that means it should not be called
+     * anymore.
```

Comment:
On one line, I suggest using a shorter one: `Whether this definition is deprecated or not.`